### DISCUSSION TITLE
Using Bootstrap-vue for inertia

### DISCUSSION THREAD

**Discussion by tsommie** [nascent-africa/jetstrap#35] [Ideas]
I've been thinking, would it be a good idea to use Bootstrap-Vue for inertia stubs?

**Comment by AdamGaskins** [comment]
We started using bootstrap-vue two years ago in a project at work. The main issue we've had is that we can't upgrade to Vue 3 because bootstrap-vue is stuck on Vue 2, and [probably will be for a while](https://github.com/bootstrap-vue/bootstrap-vue/issues/5196). Just something to consider

Love that this package exists though! Thanks for all the work.
